rancher/v1.2/en/cattle/adding-load-balancers/index.md

```diff
@@ -93,9 +75,9 @@ domain.com.* -> hdr_beg(host) -i domain.com.
 ```
 <br>
 
-**Priority with Multiple Hostname Routing Rules**
+**Priority with Multiple Port Rules**
 
-When there are multiple hostname routing rules, the order of priority is as follows:
+When there are multiple port rules for the same service, the order of priority is as follows:
```

Comment:
we also allow specifying custom priorities (numbered) via API; it takes precedence over default ordering. UI might not expose it for this release, but it is available via API/compose